docker-compose.yml

```diff
@@ -6,12 +6,16 @@ services:
       context: .
       dockerfile: Dockerfile-mysterium-node
     container_name: mysterium-node
+    cap_add:
+      - NET_ADMIN
```

Comment:
privileged is not good approach, would recommend it for debug. it also didn't work for me if I create tun in RUN step on build, with entrypoint it works.